.gitignore

```diff
@@ -20,7 +20,7 @@ SAFETY_LOCK
 **/**.network
 **/**.target
 **/**.qcow2
-**/test.py
+**/test*.py
```

Comment:
This might bite us if we add pytest tests and happen to use this naming convention.

Reply:
Good catch!